app/views/spree/admin/products/_seo_form.html.haml

```diff
@@ -1,11 +1,11 @@
 .row{"data-hook" => "admin_product_meta_form"}
   .alpha.eleven.columns
     = f.field_container :meta_description do
-      = f.label :meta_keywords, t(:meta_keywords)
+      = f.label :meta_keywords, t(:product_search_keywords)
       %br/
       = f.text_field :meta_keywords, :class => 'fullwidth', :rows => 6
     = f.field_container :meta_description do
-      = f.label :meta_description, t(:meta_description)
```

Comment:
Is there a reason why we can't simply replace the translations of these two terms?

Reply:
Hi @sauloperez , I thought that you had the wrong file, because in the one I changed the meta_description label has already changed to 'SEO_keywords', which I added on the  en.yml file to be translated, not sure now if you meant another thing?

Reply:
> @lin-d-hop and I agreed to change the labels to add more readability to the code

In that case, can you check whether the previous keys are used at all and can be removed?

Reply:
For this reason we changed them, because there were not keys yet introduced on the en.yml other any other local yml file. I couldn't find these keys in the whole project, just in the form ;) (what actually let me a little bit surprised )

Reply:
aha! good catch!